internal/pkg/skuba/upgrade/cluster/versions.go

```diff
@@ -106,5 +106,13 @@ func UpgradePath() ([]*version.Version, error) {
 	if err != nil {
 		return []*version.Version{}, err
 	}
+	upgradePath, err := upgradePathWithCurrentClusterVersion(currentClusterVersion)
+	if err != nil {
+		return []*version.Version{}, err
+	}
+	return upgradePath, nil
+}
+
+func upgradePathWithCurrentClusterVersion(currentClusterVersion *version.Version) ([]*version.Version, error) {
```

Comment:
I don't understand this change, the logic in `UpgradePathWithAvailableVersions` was already doing this, no?, what is the intent of this changeset?

Reply:
i wanted to drop this, it was still from experimenting with another way of doing it...